collectors/python.d.plugin/tomcat/tomcat.chart.py

```diff
@@ -104,10 +105,12 @@ def _get_data(self):
         data = None
         raw_data = self._get_raw_data()
         if raw_data:
+            # Fix attributes ending with single quote content
+            raw_data = re.sub(r"='([^']+)'([^']+)''", r"='\g<1>\g<2>'", raw_data)
```

Comment:
please compile the regex

Reply:
and add some comments why we need it (comments + link the the issue would be nice)